client/app/templates/Client/edit.html.twig

```diff
@@ -13,7 +13,7 @@
 {% block breadcrumbs %}
     {{ macros.breadcrumbsArray([
         { href: url('homepage'), text: 'Your reports' },
-        { href: url('client_show', {clientId: client.id}), text: client.firstname ~ '\'s details' }
+        { href: url('deputyship_details_clients'), text: 'Your clients' },
```

Comment:
Generally all the text is templated. Should we be using straight text like Your clients?